src/adaptors/lynex-fusion/index.js

```diff
@@ -60,7 +59,7 @@ const getApy = async () => {
     );
     const poolMeta = `${found?.type}`
     const symbol = found
```

Comment:
.

Reply:
Why is there a discrepancy between code on defillama's yield-server repo and UI? Pool "sAMM-alUSD-USDC" is shown with a "(undefined)" on the UI but according to code that shouldn't happen. It's fixed apparently with the PR as seen on the test suites results.
Or are the parenthesis displaying the poolMeta?

Reply:
> Or are the parenthesis displaying the poolMeta?

yes

Reply:
@Vasuveda all clear now?

Reply:
Yes! Thank you for clarification. Will fix it soon then.